.lgtm.yml

```diff
@@ -1,3 +1,2 @@
 queries:
-  - exclude: py/missing-equals
-  - exclude: py/conflicting-attributes
+  - exclude: py/missing-equals,py/conflicting-attributes
```

Comment:
@jab The original syntax looks good to me. The problem is that lgtm.com uses the `lgtm.yml` file from the master branch when filtering excluded queries. As a result the `exclude:` lines above will only take effect after this pull-request has been merged.

Reply:
Thanks @aibaars! (And on the off chance it wouldn't be too complicated to add at some point, it'd be cool if lgtm could use the lgtm.yml present in the revision it's testing rather than always using the one from master.) Thanks again, much appreciate your finding your way here to check my syntax!